xbmc/cores/VideoPlayer/DVDSubtitles/DVDSubtitlesLibass.cpp

```diff
@@ -316,105 +364,106 @@ void CDVDSubtitlesLibass::ApplyStyle(const std::shared_ptr<struct KODI::SUBTITLE
     }
 
     // It is mandatory set the FontName, the text is case sensitive
-    style->FontName = strdup(subStyle->fontName.c_str());
-
-    if (m_subtitleType != NATIVE || subStyle->assOverrideStyles != OverrideStyles::POSITIONS)
+    free(style->FontName);
+    if (subStyle->fontName == FONT_PLACEHOLDER_FAMILYNAME)
```

Comment:
Not a big fan of `FONT_PLACEHOLDER_FAMILYNAME` as it sounds like an hack, which this is not. Why not compare to m_defaultFontFamilyNamedirectly?

Reply:
how do you compare with m_defaultFontFamilyNamedirectly?
FONT_PLACEHOLDER_FAMILYNAME is needed to detect if the default kodi font is set in the setting